src/toil/__init__.py

```diff
@@ -141,9 +140,9 @@ def lookupEnvVar(name, envName, defaultValue):
         return value
 
 
-def logProcessContext(config, logger):
+def logProcessContext(config):
     # toil.version.version (string) canont be imported at top level because it conflicts with
     # toil.version (module) and Sphinx doesn't like that.
     from toil.version import version
-    logger.info("Running Toil version %s.", version)
-    logger.info("Configuration:\n %s", pformat(config.__dict__))
+    log.info("Running Toil version %s.", version)
+    log.info("Configuration: %s", config.__dict__)
```

Comment:
One thing, can you change this line to debug? You don't have to squash/rebase the commit to change this